### PR TITLE
Css Basic task

### PR DESCRIPTION
**Pull request by Artem781**
Please review. Basic task adaptation site.

### CODE REVIEW

**Comment by pluhin** [comment]
You have conflict in this file
Conflicting files
anaron/02.CSS/CSS_02_Cards_V1.html

File: anaron/02.CSS/CSS_02_Cards_V2.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<HTML>
+
+<HEAD>
+    <meta charset="utf-8">
+    <TITLE>Спрайты</TITLE>
+</HEAD>
+<style>
+    .three {
```
Comment (pluhin) [inline anaron/02.CSS/CSS_02_Cards_V2.html]: Here css code can be shorter

File: anaron/02.CSS/CSS_02_Cards_V2.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        width: 136px;
+        height: 180px;
+        display: inline-block;
+        margin-left: -40px
+    }
+
+</style>
+
+<BODY>
+    <div class="three" img="cards.png"></div>
```
Comment (pluhin) [inline anaron/02.CSS/CSS_02_Cards_V2.html]: what is img attribute?

File: anaron/03.CSS/06_CSS_Adaptation_Aditional.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+        }
+
+        li {
+            position: relative;        /* понедельник и тд */
+        }
+
+        li ul {                        /* подменю */
+            display:block;
+            position: absolute;
+            top: 26px;
```
Comment (pluhin) [inline anaron/03.CSS/06_CSS_Adaptation_Aditional.html]: here it is better to use auto instead 26px

File: anaron/03.CSS/06_CSS_Adaptation.html [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+        }
+
+        li {
+            position: relative;
+        }
+
+        li ul {
+            display: none;
+            position: absolute;
+            top: 26px;
```
Comment (pluhin) [inline anaron/03.CSS/06_CSS_Adaptation.html]: here better to use auto value instead 26px,

File: anaron/02.CSS/CSS_02_Cards_V2.html (hunk 1 above)
Comment (Artem781) [reply anaron/02.CSS/CSS_02_Cards_V2.html]: yes. thank you) i fixed.

File: anaron/03.CSS/06_CSS_Adaptation_Aditional.html (hunk 3 above)
Comment (Artem781) [reply anaron/03.CSS/06_CSS_Adaptation_Aditional.html]: corrected. Thank you

File: anaron/03.CSS/06_CSS_Adaptation.html (hunk 4 above)
Comment (Artem781) [reply anaron/03.CSS/06_CSS_Adaptation.html]: fixed) Thank you)

**Pull request merged**